java/test/jmri/jmrit/operations/rollingstock/OperationsLoggerTest.java

```diff
@@ -1,76 +0,0 @@
-package jmri.jmrit.operations.rollingstock;
-
-import java.io.File;
-import java.util.Locale;
-import jmri.InstanceManager;
-import jmri.jmrit.operations.OperationsTestCase;
-import jmri.jmrit.operations.locations.Location;
-import jmri.jmrit.operations.locations.Track;
-import jmri.jmrit.operations.rollingstock.cars.Car;
-import jmri.jmrit.operations.rollingstock.cars.CarManager;
-import jmri.jmrit.operations.rollingstock.cars.CarTypes;
-import jmri.jmrit.operations.setup.Setup;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-/**
- * Tests for the Operations Logger class
- *
- * @author	Dan Boudreau Copyright (C) 2010
- *
- */
-public class OperationsLoggerTest extends OperationsTestCase {
-
-    // test creation
-    @Test
-    public void testCreate() {
-        // load a car
-        CarManager manager = InstanceManager.getDefault(CarManager.class);
-        Car c1 = manager.newCar("CP", "1");
-        CarTypes ct = InstanceManager.getDefault(CarTypes.class);
-        ct.addName("Boxcar");
-        c1.setTypeName("Boxcar");
-        c1.setLength("40");
-        // turn on logging
-        Setup.setCarLoggerEnabled(true);
-        // log is created after a car is placed
-        File file = new File(InstanceManager.getDefault(RollingStockLogger.class).getFullLoggerFileName());
```

Comment:
Is creation of this file verified elsewhere? If it isn’t, this test should be restored and modified to use a concrete class.